packages/app/src/index.ts

```diff
@@ -46,18 +46,20 @@ const createFlamelinkFactory: App.FactoryCreator = () => {
       precache: typeof config.precache === 'undefined' ? true : config.precache,
       proxySupported: typeof Proxy !== 'undefined',
       usesAdminApp: isAdminApp(config.firebaseApp),
-      isNodeEnvironment: isNodeEnvironment()
+      isNodeEnvironment: isNodeEnvironment(),
     }
 
     ensureValidContext(context)
 
     initRegisteredModules(context)
 
     const api: App.PublicApi = PUBLIC_MODULES.reduce((acc: any, moduleName) => {
-      return Object.assign(acc, {
-        get [moduleName]() {
+      return Object.defineProperty(acc, moduleName, {
+        enumerable: true,
```

Comment:
For some reason, the newer version of Rollup transpiled the `Object.assign` into a version of `Object.defineProperty` with `enumerable: false` which caused the generated flamelink app instance to be an empty object.